src/nblib/nblib/bondtypes.h

```diff
@@ -45,7 +45,7 @@
 #ifndef GMX_NBLIB_BONDTYPES_H
 #define GMX_NBLIB_BONDTYPES_H
 
-#include "gromacs/nblib/particletype.h"
+#include "nblib/nblib/particletype.h"
```

Comment:
Do we want nblib/nblib or just nblib?

Reply:
We should look at the way it works for gmxapi and see if we like that. I am relatively sure we would not want the headers or source files in the top level directory of nblib so the main question would be if we want nblib/nblib or nblib/someOtherName.